modules/classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
```

Comment:
- [ ] Kindly make sure that the relationship between classroom and student is created i.e. when adding a student to a classroom it also sets the classroom for the student 👉🏽 `attr_reader :students`